src/logger.js

```diff
@@ -1,8 +1,12 @@
 // A simple logging function
 export default class Logger {
   constructor(system, verbose) {
-    this.system = system;
-    this.verbose = verbose;
+    if(!Logger.instance) {
```

Comment:
CS: Only a matter of preferences, but I have seen that the files are written with a "space" character between the ```if clause``` and the ```open parenthesis```

Reply:
This should be caught by using Prettier.